cmd/kubeadm/app/phases/upgrade/staticpods.go

```diff
@@ -366,7 +346,7 @@ func performEtcdStaticPodUpgrade(client clientset.Interface, waiter apiclient.Wa
 
 	// Checking health state of etcd after the upgrade
 	fmt.Println("[upgrade/etcd] Waiting for etcd to become available")
-	if _, err = newEtcdClient.WaitForClusterAvailable(podRestartDelay, retries, retryInterval); err != nil {
+	if _, err = newEtcdClient.WaitForClusterAvailable(noDelay, retries, retryInterval); err != nil {
```

Comment:
We can also cleanup the delay parameter of `WaitForClusterAvailable`. It's only used for this case and is not meaningful otherwise, so we can safely reduce the clutter there too.

Reply:
Yes，I agree

Reply:
Updated and thanks for reviewing